WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -56,7 +56,9 @@ class AbstractPostListViewController : UIViewController, WPContentSyncHelperDele
     
     
     var postListFooterView : PostListFooterView!
-    var animatedBox : WPAnimatedBox?
+    private var animatedBox : WPAnimatedBox = {
```

Comment:
It looks like the animated box is never modified, so this can be made a constant. You can also just initialise it directly: `private let animatedBox = WPAnimatedBox.newAnimatedBox()`.